python/ray/dataframe/io.py

```diff
@@ -260,3 +261,212 @@ def read_csv(filepath,
         df_obj_ids.append(df)
 
     return DataFrame(df_obj_ids, columns)
+
+
+def read_json(path_or_buf=None,
+              orient=None,
+              typ='frame',
+              dtype=True,
+              convert_axes=True,
+              convert_dates=True,
+              keep_default_dates=True,
+              numpy=False,
+              precise_float=False,
+              date_unit=None,
+              encoding=None,
+              lines=False,
+              chunksize=None,
+              compression='infer'):
+
+    warnings.warn("Defaulting to Pandas implementation",
+                  PendingDeprecationWarning)
+
+    port_frame = pd.read_json(path_or_buf, orient, typ, dtype,
+                              convert_axes, convert_dates, keep_default_dates,
+                              numpy, precise_float, date_unit, encoding,
+                              lines, chunksize, compression)
+    ray_frame = ray.dataframe.from_pandas(port_frame, get_npartitions())
```

Comment:
This has to change to `ray.dataframe.utils.from_pandas`.

Also, I prefer this as an import: `from .utils import from_pandas`.